packages/admin/resources/views/components/stats/card.blade.php

```diff
@@ -117,9 +119,9 @@
                 },
 
                 updateChart: function () {
-                    this.chart.data.labels = this.labels
-                    this.chart.data.datasets[0].data = this.values
-                    this.chart.update()
+                    chart.data.labels = this.labels
```

Comment:
Does this actually still work? How does it access the chart variable without a property on the object?

Reply:
Yes it does work. I'm not able to explain why since I'm not that experienced with alpine. All I know is that it wasn't working when I started and it ended op working with the proposed changes.

Reply:
Could you double check it works when polling is used with the chart? So the chart data changes data

Reply:
Yes it does. Maybe you could check it too.

Reply:
I'm quite sure this doesn't work and broke charts actually...